bin/compile

```diff
@@ -93,6 +93,11 @@ fi
 
 speak "Running stack"
 cd "$WORK_DIR"
+# Makefile support (if you need other build tools than Stack that you need to use in conjunction)
+if [ -f $BUILD_DIR/Makefile ]; then
+    speak "Makefile detected"
+    make -C $BUILD_DIR
+fi
```

Comment:
Great! In that case, should the call be
```
make -C "$BUILD_DIR"
make install -C "$BUILD_DIR"
```
instead of just the
```
make -C "$BUILD_DIR"
```
line?